presto-hive/bin/run_hive_glue_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I don't think this script is needed as it simply launched maven with selected profile. This can be done within Travis yaml directly.

Reply:
Sure, I will remove this script.